src/schema/v2/image/normalize.ts

```diff
@@ -36,7 +36,7 @@ export const setVersion = (
 
 const normalizeImageUrl = (image) => {
   const image_url = grab(image, ["url", "image_url"])
-  if (!image_url) return null
+
```

Comment:
When the image is created,  the is no `imageURL` until Gemini has processed the image. To return the images from the mutation we therefor shouldn't filter images anymore.

Reply:
**praise:** Nice, good catch.

Reply:
**suggestion:** I am wondering if we should only enable this only when a flag is specified so the behavior is still the same in the frontend

Reply:
I mean something like:
```typescript
  if (!image_url && !includeProcessing ) return null
```

Reply:
I understand though that this is not probably necessarily needed since we don't use returned images anywhere now but being explicit about it might be useful

Reply:
+1 to @MounirDhahri 's suggestion, especially if we are using normalizeImageUrl elsewhere besides handling MyCollectionImages.

Reply:
You're right. I'll double-check if this could lead to issues in other places. Probably, it's a good idea to add the `includeProcessing` parameter (or `includeAll` - failed images wouldn't have an `imageURL` either) because we only need to show these images in the MyCollection context.